go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/neuropoly/bids-hook
 
-go 1.18
+go 1.19
```

Comment:
This is to get url.JoinPath(). I think this is fair now, since Gitea itself requires this now.

It does make developing on Ubuntu LTS annoying. Which is why before I nagged you to [downgrade](https://github.com/neuropoly/bids-hook/pull/1). But now I had to bite the bullet to deal with Gitea anyway, so to get around that, I [used this instead](https://go.dev/doc/manage-install)

```
go install golang.org/dl/go1.19.5@latest
go1.19.5 download
cat >> ~/.profile <<EOF
# set PATH so it includes go
if [ -d "$HOME/go/bin" ] ; then
    PATH="$HOME/go/bin:$PATH"
fi
EOF
ln -s go1.19.5 ~/go/bin/go
```